### ISSUE TITLE
Allow photometric models to be initialized with albedos

### ISSUE THREAD

**Issue by mommermi** [NASA-Planetary-Science/sbpy#146]
For `sbpy.thermal` it would be great to have functionality in `sbpy.photometry` such that, e.g., `HG` models can be initialized with a geometric albedo or Bond albedo in order to calculate the target's radius (with H provided) or H (with radius provided).

**Comment by mkelley** [comment]
Or, what about a `.to_phys()` method that would convert the HG model to a `Phys` object?  D and Ap could be added to the `Phys` object if appropriate keyword arguments were given:
```python
>>> p = hg.to_phys(Ap=0.04)
>>> print(p['D'], p['Ap'])
100 km 0.04
```

**Comment by jianyangli** [comment]
The functionality to convert between magnitude, geometric albedo, and radius is implemented in an equivalency function `reflectance` in PR #136 , which I haven't finalize yet.

But I do like the idea of adding a `.to_phys()` method to capture everything that can be calculated with a `HG` or other model class object into a `Phys` object.

**Comment by mommermi** [comment]
Yes, `.to_phys()` sounds like a great idea - I will steal this for `sbpy.thermal` :)

Thanks, @jianyangli , no rush. I just wanted to add it to the list of things that are to be done.

**Comment by jianyangli** [comment]
Addressed by #213
